src/ILCompiler.ReadyToRun/src/JitInterface/CorInfoImpl.ReadyToRun.cs

```diff
@@ -65,7 +67,18 @@ public GenericContext(TypeSystemEntity context)
 
         public override int GetHashCode() => Context.GetHashCode();
 
-        public override string ToString() => Context.ToString();
+        public void AppendMangledName(NameMangler nameMangler, Utf8StringBuilder sb)
+        {
+            StringBuilder output = new StringBuilder();
```

Comment:
Unused? (Also the `System.Text` using above.)

Reply:
Thanks for spotting, removed in 2nd commit.